src/modules/app/containers/App.js

```diff
@@ -10,24 +12,30 @@ import PlatformProvider from '../../platform/containers/PlatformProvider'
 import routesMap from '../routesMap'
 import Switcher from './Switcher'
 import theme from '../constants/theme'
+import { Store } from 'redux'
+
+export interface PropsType {
```

Comment:
Do we need empty PropTypes now?
Should we always export them?

Reply:
I think we should not export them. Not sure about the empty ones.